ivadomed/postprocessing.py

```diff
@@ -0,0 +1,205 @@
+# Deals with postprocessing on generated segmentation.
+
+import numpy as np
+import nibabel as nib
+from scipy.ndimage.measurements import label
+from scipy.ndimage.morphology import binary_fill_holes
+
+
+def threshold_predictions(predictions, thr=0.5):
+    """Threshold predictions.
+
+    Threshold a soft (ie not binary) array of predictions given a threshold value, and returns
+    a binary array.
+
+    Args:
+        predictions (array): array to binarise.
+        thr (float): Threshold value: voxels with a value < to thr are assigned 0 as value, 1
+            otherwise.
+    Returns:
+        array: Array containing only zeros or ones.
+
+    """
+    thresholded_preds = np.copy(predictions)[:]
+    low_values_indices = thresholded_preds < thr
+    thresholded_preds[low_values_indices] = 0
+    low_values_indices = thresholded_preds >= thr
+    thresholded_preds[low_values_indices] = 1
+    return thresholded_preds.astype(np.int)
+
+
+def threshold_predictions_nib(nib_predictions, thr=0.5):
+    """Threshold predictions from a nibabel object.
+
+    Threshold a soft (ie not binary) array of predictions from a nibabel object given a threshold value,
+    and returns a nibabel object with binary data.
+    Note: this function calls threshold_predictions().
+
+    Args:
+        nib_predictions (nibabelObject): nibabel image to binarise.
+        thr (float): Threshold value: voxels with a value < to thr are assigned 0 as value, 1
+            otherwise.
+    Returns:
+        nibabelObject: Nibabel object containing only zeros or ones.
+
+    """
+    data = nib_predictions.get_data()
```

Comment:
it is [now recommended](https://nipy.org/nibabel/reference/nibabel.dataobj_images.html?highlight=get_fdata#nibabel.dataobj_images.DataobjImage.get_data) to use `fget_data` due to future deprecation

Reply:
Changing for `get_fdata` (instead of `get_data` and `fget_data`).